tests_e2e/lisa/runbook/azure.yml

```diff
@@ -0,0 +1,82 @@
+name: azure
+extension:
+  - "../testsuites"
```

Comment:
Is this the directory all of the LISA tests need to be in? Or will LISA look for tests in the entire /lisa directory

Reply:
The LISA runbook points to the test location

```
extension:
  - "../testsuites"

```